ui/app/AppLayouts/Chat/panels/GroupChatPanel.qml

```diff
@@ -0,0 +1,180 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.13
+
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+
+
+RowLayout {
+     id: root
+
+     property var sectionModule
+     property var chatContentModule
+     property var rootStore
+     property int maxHeight
+
+     signal panelClosed()
+
+     QtObject {
+         id: d
+
+         property ListModel groupUsersModel: ListModel { }
+         property ListModel contactsModel: ListModel { }
+         property var addedMembersIds: []
+         property var removedMembersIds: []
+
+         function initialize () {
+             groupUsersModel.clear()
+             contactsModel.clear()
+             addedMembersIds = []
+             removedMembersIds = []
+             tagSelector.namesModel.clear()
+         }
+     }
+
+     ListView {
+         id: groupUsersModelListView
+         visible: false
+         model: root.chatContentModule.usersModule.model
+         delegate: Item {
+             property string publicId: model.id
+             property string name: model.name
+             property string icon: model.icon
+             property bool isAdmin: model.isAdmin
+         }
+     }
+
+     ListView {
+         id: contactsModelListView
+         visible: false
+         model: root.rootStore.contactsModel
+         delegate: Item {
+             property string publicId: model.pubKey
+             property string name: model.name
+             property string icon: model.icon
+         }
+     }
+
+     clip: true
+
+     Component.onCompleted: {
+         d.initialize()
+
+         // Build groupUsersModel type from model type (to fit with expected StatusTagSelector format
+         for (var i = 0; i < groupUsersModelListView.count; i ++) {
+             var entry = groupUsersModelListView.itemAtIndex(i)
+
+             // Add all group users different than me
+             if(!entry.isAdmin) {
+                 d.groupUsersModel.insert(d.groupUsersModel.count,
+                                      {"publicId": entry.publicId,
+                                       "name": entry.name,
+                                       "icon": entry.icon})
+             }
+         }
+
+         // Build contactsModel type from model type (to fit with expected StatusTagSelector format
```

Comment:
> Building proxy models imperatively in qml is a bad idea, mainly because of source model synchronization issues. If the property of item changes in the source model it will not be reflected in the manually built proxy model, e.g. if for some reason icon is changed in the `root.rootStore.contactsModel` it will not be reflected in `d.contactsModel`, etc.

An issue was created here to keep track of that: https://github.com/status-im/status-desktop/issues/5956